src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -133,6 +165,7 @@ void syncSubscriptions(String orgId, int offset, int limit) {
     int pageSize = limit + 1;
     List<Subscription> subscriptions =
         subscriptionService.getSubscriptionsByOrgId(orgId, offset, pageSize);
+    log.info("Got subscriptions of size {} from external service ", subscriptions.size());
```

Comment:
I fully admit this is the nitpickiest of comments: It may read "easier" if applying the previously mentioned splunk standards, and listing the org would be good too: "Fetched numSubs={} for orgId={} from external service."

OTOH, this same information and phrasing is found further down on line 178, so maybe its okay anyway.

Reply:
Updated.